packages/gateway/src/features/registration/utils.ts

```diff
@@ -63,3 +69,20 @@ export async function setCollectorForPrintInAdvance(
 
   return details
 }
+
+export function getPractitionerRoleFromPractitioner(
+  practitioner: SavedPractitioner,
+  bundle: SavedBundle<Resource>
+) {
+  const bundleEntries = bundle.entry
+  for (const entry of bundleEntries) {
+    if (
+      isPractitionerRole(entry.resource) &&
+      //@ts-ignore
```

Comment:
What is typescript's complain here? :smiley:

Reply:
Not really using it now. Removing this.